docs/documentation/securing_apps/topics/saml/java/jboss-adapter/jboss_adapter_installation.adoc

```diff
@@ -70,6 +70,8 @@ ifeval::[{project_product}==true]
 $ cd $JBOSS_HOME
 $ ./bin/jboss-cli.sh -c --file=bin/adapter-elytron-install-saml.cli
 ----
++
+NOTE: https://access.redhat.com/articles/2026253#EAP_74[EAP supports OpenJDK 17 and Oracle JDK 17] since 7.4.CP7 and 7.4.CP8 respectively. But note that the new java version makes the elytron variant compulsory (do not use the legacy adapter with JDK 17) and, after running the adapter CLI file, the EAP provided `enable-elytron-se17.cli` script should be executed. Both scripts are necessary, in that order, to configure the elytron adapter and then remove the incompatible EAP subsystems. Check https://access.redhat.com/articles/6956863[the following EAP article] for more information.
```

Comment:
```suggestion
NOTE: https://access.redhat.com/articles/2026253#EAP_74[EAP supports OpenJDK 17 and Oracle JDK 17] since 7.4.CP7 and 7.4.CP8 respectively. Note that the new java version makes the elytron variant compulsory, so do not use the legacy adapter with JDK 17. Also, after running the adapter CLI file, execute the `enable-elytron-se17.cli` script provided by EAP. Both scripts are necessary to configure the elytron adapter and remove the incompatible EAP subsystems. For more details, see this https://access.redhat.com/articles/6956863[Security Configuration Changes] article.
```

Reply:
Done!